app/assets/stylesheets/administrate/components/_date_time_picker.scss

```diff
@@ -1,3 +1,11 @@
 .form-field {
   position: relative;
 }
+
+.bootstrap-datetimepicker-widget{
+  table{
+    td, th{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space